x-pack/plugins/monitoring/public/alerts/cpu_usage_alert/cpu_usage_alert.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { AlertTypeModel } from '../../../../triggers_actions_ui/public/types';
+// import { Expressions } from './components/expression';
+import { validate } from './validation';
+import { ALERT_CPU_USAGE } from '../../../common/constants';
+import { Expression } from './expression';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { CpuUsageAlert } from '../../../server/alerts';
```

Comment:
Maybe it's worth adding `paramDetails` somewhere in `x-pack/plugins/monitoring/common/...`, so we don't have to get stuff from `server` and bypass linting

Reply:
Hmm. What do you mean by "bypassing linting"? My impression is the line above this just ignores one specific linting rule, but does apply the rest of the linting rules?

Reply:
I think the rule/intent is that `server` and public `apps` should be completely independent from one another and can only gather/import types/resources from a centralized location (like their parent or .../common), but not from each other. This is to avoid any stateful logic/dependancies. So, I figured maybe https://github.com/elastic/kibana/blob/0bbc150e89e678f8fbf44b88ac43fae4aa83dae5/x-pack/plugins/monitoring/server/alerts/cpu_usage_alert.ts#L56 can live somewhere outside (or obtained via an API call) if frontend might also rely on it. But, maybe not a big deal here